scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -86,7 +86,7 @@ BO_ 206 DCM_TORQUE_REQUEST : 4 DCM
 SG_ Torque_Request : 0|32@1+ (1,0) [-3.4E038|3.4E038] "" LOGGER
 
 BO_ 207 DCM_CRITICAL_ERRORS: 8 DCM
-SG_ DUMMY_CRITICAL_ERROR : 0|1@1+ (1,0) [0|1] "" DEBUG
+SG_ DUMMY : 0|1@1+ (1,0) [0|1] "" DEBUG
```

Comment:
Had to update the names after using codegen so the executable compiles without errors

Reply:
Wait, why? I'm confused on how this fixed anything?

Reply:
So previously we had 

```
SG_ DUMMY_CRITICAL_ERROR ...
```
and the corresponding enum should have been

```
BOARD_CRITICAL_ERROR_DUMMY_CRITICAL_ERROR
```
but instead I (manually) wrote
```
BOARD_CRITICAL_ERROR_DUMMY
```

Now I could either update all instances of `BOARD_CRITICAL_ERROR_DUMMY` to `BOARD_CRITICAL_ERROR_DUMMY_CRITICAL_ERROR` or change `SG_ DUMMY_CRITICAL_ERROR` to `SG_ DUMMY` as a workaround, I chose the latter option. 

TL;DR I hand wrote the enum poorly previously, which conflicts with what the codegen outputs.